### PR TITLE
Added Front end and functionality

### PR DESCRIPTION
**Pull request by Cwagne17**
I implemented the front end.
Connected the back-end.

### CODE REVIEW

File: api/src/app.ts [hunk 1]
```diff
@@ -48,8 +50,14 @@ app.patch("/", function(req,res){
 
   //Deletes a Task
 app.delete("/delete/:taskId", async function(req,res){
-  await taskDataStore.deleteTask(req.params.taskId)
-  res.sendStatus(200);
+  console.log(req.params.taskId);
```
Comment (pzalep1) [inline api/src/app.ts]: :fire: console.log.

File: api/src/app.ts [hunk 2]
```diff
@@ -48,8 +50,14 @@ app.patch("/", function(req,res){
...
+  try{
+    await taskDataStore.deleteTask(req.params.taskId)
+    res.sendStatus(200);
+  }catch (err){
+    console.log(err);
```
Comment (pzalep1) [inline api/src/app.ts]: This should just send the error. Usually there would be some processing of the error itself to give better context to the user, but for the purposes of this project just returning the error to the client would be sufficient.

**Review by pzalep1 — CHANGES_REQUESTED** [review]
Good work on this! Just two small changes from me.

File: TakeHomeTest/src/app/web-request.service.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WebRequestService {
+
+  readonly ROOT_URL;
+
+  constructor(private http: HttpClient) {
+    this.ROOT_URL = "http://localhost:3000";
```
Comment (EVogel1999) [inline TakeHomeTest/src/app/web-request.service.ts:12]: This is a really good method that we also use in both CARD and CLARK!  It's great for when we need to reference our deployed API versus our local development API.

**Review by EVogel1999 — APPROVED** [review]
Great work Chris!

**Review by pzalep1 — APPROVED** [review]
Good work on this!

**Pull request merged**